dask/base.py

```diff
@@ -1362,14 +1361,25 @@ def get_scheduler(get=None, scheduler=None, collections=None, cls=None):
         elif isinstance(scheduler, str):
             scheduler = scheduler.lower()
 
+            try:
+                from distributed import default_client
```

Comment:
I've not tried it out locally yet, but just from reading the diff here it looks like this might raise an `ImportError` when `distributed` isn't installed. Maybe I'm missing something though

Reply:
Ah, it looks like this is showing up in our mindeps CI builds

Reply:
yes, was already concerned about this. should be an easy fix